src/Components/Server/src/Circuits/CircuitFactory.cs

```diff
@@ -45,7 +45,7 @@ internal class CircuitFactory
             ClaimsPrincipal user,
             IComponentApplicationStateStore store)
         {
-            var scope = _scopeFactory.CreateScope();
+            var scope = _scopeFactory.CreateAsyncScope();
```

Comment:
Where is this disposed?

Reply:
CircuitHost is responsible for it, it had special case logic for AsyncDisposable, it's updated now as well